code/game/objects/buckling.dm

```diff
@@ -42,12 +42,16 @@
 	M.update_canmove()
 	post_buckle_mob(M)
 	M.throw_alert("buckled", new_master = src)
-	if(burn_state == 1) //Sets the mob on fire if you buckle them to a burning object
+
+	return 1
+
+/obj/buckle_mob(mob/living/M)
+	if(burn_state == 1) //Sets the mob on fire if you buckle them to a burning atom/movableect
 		M.adjust_fire_stacks(1)
 		M.IgniteMob()
```

Comment:
the parent need to be called first and this fire code only need to be run if the parent returns 1.